cassandra-3/pom.xml

```diff
@@ -45,8 +45,19 @@
                     <groupId>org.slf4j</groupId>
                     <artifactId>log4j-over-slf4j</artifactId>
                 </exclusion>
+                <!-- Exclude bundled version that is not compatible with JDK names that do not have dots in their names -->
+                <exclusion>
+                    <groupId>com.github.jbellis</groupId>
+                    <artifactId>jamm</artifactId>
+                </exclusion>
             </exclusions>
         </dependency>
+        <!-- Fixes compatibility JDK versions that may not yet dots in their names -->
+        <dependency>
+            <groupId>com.github.jbellis</groupId>
+            <artifactId>jamm</artifactId>
+            <version>0.3.3</version>
```

Comment:
Should not this go into BOM instead?

Reply:
I choose not to do that because the only one I wanted to override was cassandra 3 as the other two modules are unaffected.  If you want it defined in the BOM I can do that, just let me know.